WalletWasabi/WabiSabi/Client/CoinJoin/Manager/CoinJoinManager.cs

```diff
@@ -211,8 +211,8 @@ async Task<IEnumerable<SmartCoin>> SanityChecksAndGetCoinCandidatesFunc()
 						throw new CoinJoinClientException(CoinjoinError.AllCoinsPrivate);
 					}
 
-					// If coin candidates are already private and the user doesn't override the StopWhenAllMixed, then don't mix.
-					if (coinCandidates.All(x => !x.Confirmed || x.IsPrivate(walletToStart.AnonScoreTarget)) && startCommand.StopWhenAllMixed)
```

Comment:
`startCommand.StopWhenAllMixed` is false the if condition cannot be satisfied - no matter what. It is false if `AutoCoinJoin` is turned **ON**. This is why the change will only affect AutoCoinJoin.